force-app/main/default/lwc/platformDevCertCalculator/platformDevCertCalculator.html

```diff
@@ -0,0 +1,90 @@
+                onchange={handleChange}
+            ></lightning-input>
+            <lightning-input 
+                type="number" 
+                name="procAutoLogic" 
+                label="Process Automation and Logic: 30% " 
+                style="color:blue"
+                value={procAutoLogicScore}
+                onchange={handleChange}
+            ></lightning-input>
+            <lightning-input 
+                type="number" 
+                name="userInterface" 
+                label="User Interface: 25% " 
+                style="color:blue"
+                value={userInterfaceScore}
+                onchange={handleChange}
+            ></lightning-input>
+            <lightning-input 
+                type="number" 
+                name="testDebugDeployment" 
+                label="Testing, Debugging and Deployment: 22% " 
+                style="color:blue"
+                value={testDebugDeploymentScore}
+                onchange={handleChange}
+            ></lightning-input>
+        </div>
+        <!-- Display Results -->
+        <div slot="footer">
+            <h1 class="slds-text-heading_large slds-text-title_caps resultText">Certification Results</h1>
+            <p class="slds-text-heading_medium">Certification Score: <span class="slds-text-color_destructive">{certificationScore}%</span></p>
+            <div>
+                <h1 class="slds-text-heading_large">Previous Attempts</h1>
+                <lightning-tabset>
+                    <div for:each={attemptHistory} for:item="attempt" key={attempt.Id}>
+                        <lightning-tab label={attempt.Id}>
+                            <c-score-information
+                                score={attempt.Score}
+                                attempt-id={attempt.Id}
+                                number-of-questions={numberOfQuestions}
+                                ondeleteattempt={deleteAttemptHandler}
+                            ></c-score-information>
+                        </lightning-tab>                            
+                    </div>
+                </lightning-tabset>
+
+            </div>
+            <template lwc:if={showResources}>
+                <ul class="slds-text-align_left slds-text-heading_small slds-var-p-top_medium slds-box">
+                    <li class="resourceLi">
+                        Resources:
+                    </li>
+                    <li>
+                        <a href="https://trailhead.salesforce.com/en/credentials/platformdeveloperi">PD1 Exam Guide</a>
+                    </li>
+                    <li>
+                        <a href="https://focusonforce.com/courses/salesforce-certified-platform-deverloper-1-exams/">Focus on Force</a>
+                    </li>
+                    <li>
+                        <a href="https://courses/salesforcementor.com/p/salesforce-development-apex-fundamentals">Salesforce Mentor</a>
+                    </li>
+                </ul>
```

Comment:
Can these lit items be created in JS list or object and then be transferred into the HTML with a iterator.